### PR TITLE
Language names in audio tracks and subtitles

### PR DESCRIPTION
**Pull request by OryWareKyrie**
Change made by "Young" cat @alfablac. nothing to laugh saw. Granny 😄 https://github.com/izu-co/funimation-downloader-nx/issues/18

### CODE REVIEW

**Comment by izu-co** [comment]
Please use ESLint to meet the style requirements.

File: modules/merger.js [hunk 1]
```diff
@@ -76,9 +76,7 @@ const buildCommandMkvMerge = (videoAndAudio, onlyVid, onlyAudio, subtitles, outp
 
     args.push(`-o "${output}"`);
     args.push(
-        '--no-date',
-        '--disable-track-statistics-tags',
-        '--engage no_variable_data',
+        '--global-tags "./tag.xml"',
```
Comment (izu-co) [inline modules/merger.js:79]: Where do you get `tag.xml` from?

Comment (OryWareKyrie) [reply modules/merger.js:79]: this you can disregard. it's a global tag file that I have at the root of the script. about ESLint I'll fix it already. sorry I'm new at this :smile:

Comment (izu-co) [reply modules/merger.js:79]: Well, you would need to remove this line, since not everyone has the file.

**Comment by izu-co** [comment]
I would recommend you use the command `npx eslint . --fix` for ESLint. The current command set in the package.json is an artifact from the anidl.

File: modules/merger.js [hunk 2]
```diff
@@ -85,7 +87,7 @@ const buildCommandMkvMerge = (videoAndAudio, onlyVid, onlyAudio, subtitles, outp
                 '--video-tracks 0',
                 '--no-audio'
             );
-            args.push('--track-name 0:[Funimation]');
+            args.push('--track-name 0:[Uncut]');
```
Comment (izu-co) [inline modules/merger.js:90]: Here and in line 102 the track can also be simulcast, so assuming it's uncut is incomplete.

File: modules/merger.js [hunk 3]
```diff
@@ -97,9 +99,9 @@ const buildCommandMkvMerge = (videoAndAudio, onlyVid, onlyAudio, subtitles, outp
                 '--video-tracks 0',
                 '--audio-tracks 1'
             );
-            args.push('--track-name 0:[Funimation]');
+            args.push('--track-name 0:[Uncut]');
```
Comment (izu-co) [inline modules/merger.js:102]: See comment on line 90.

**Review by izu-co — CHANGES_REQUESTED** [review]

**Comment by izu-co** [comment]
By the way, I'm sorry that I am so picky.

**Comment by OryWareKyrie** [comment]
> By the way, I'm sorry that I am so picky.

nothing. I'm sorry I'm so NUB, it's my first time.

**Comment by izu-co** [comment]
Since you are working in your free time to support a project, there is nothing to worry about.

**Review by izu-co — APPROVED** [review]
It seems like everything is good now!

**Comment by izu-co** [comment]
Please note that this feature will be added with the next version.

**Pull request merged**